### PR TITLE
Fix name of version comparing function

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

File: node/generate-latest-dockerfile.sh [hunk 1]
```diff
@@ -131,7 +131,7 @@ for device in $devices; do
 		# Debian.
 		# For armv7hf and armv6hf, if node version is less than 4.x.x (0.10.x 0.12.x) then that image will use binaries from resin, otherwise it will use binaries from official distribution.
```
Comment (petrosagg) [inline node/generate-latest-dockerfile.sh]: Can you change this comment to reflect the code? i.e if node version is **greater or equal than 4.x.x**...

File: node/generate-all-dockerfile.sh [hunk 2]
```diff
@@ -87,7 +87,7 @@ for device in $devices; do
 
 		# For armv7hf and armv6hf, if node version is less than 4.x.x (0.10.x 0.12.x) then that image will use binaries from resin, otherwise it will use binaries from official distribution.
```
Comment (petrosagg) [inline node/generate-all-dockerfile.sh]: Can you change this comment to reflect the code? i.e if node version is **greater or equal than 4.x.x**...

File: node/generate-latest-dockerfile.sh (hunk 1 above)
Comment (nghiant2710) [inline node/generate-latest-dockerfile.sh]: @petrosagg, Updated!

**Comment by petrosagg** [comment]
SGTM

**Pull request merged**